packages/shortest/src/types/config.ts

```diff
@@ -6,4 +6,8 @@ export interface ShortestConfig {
   baseUrl: string;
   testDir: string | string[];
   anthropicKey: string;
+  mailosaur?: {
+    apiKey: string;
+    serverId: string;
+  };
```

Comment:
In order to use Email validation feature, end users need to add these configurations.